LICENSE.txt

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2024 Isaac To <candleindark@gmail.com>
```

Comment:
@yarikoptic Can you help input the correct license info. The current info is generated by Hatch mostly. I am perfectly fine the direct the license to DANDI. I just don't know the proper way to put it.

Reply:
Let's state

```suggestion
MIT License

Copyright (c) 2024 DANDI Team <info@dandiarchive.org>
```